config/default/manager_metrics_patch.yaml

```diff
@@ -1,4 +1,7 @@
-# This patch adds the args to allow exposing the metrics endpoint using HTTPS
+# This patch adds the args to allow exposing the metrics endpoint using HTTP
 - op: add
   path: /spec/template/spec/containers/0/args/0
-  value: --metrics-bind-address=:8443
+  value: --metrics-bind-address=:8080
+- op: add
+  path: /spec/template/spec/containers/0/args/1
+  value: --metrics-secure=false
```

Comment:
Can we add a TODO here and bubble up within the readme that metrics are not secure?